Source/Resources/Content/MSGraph/ConditionalAccess/Policies/Require Compliant Device for Intune Enrollment.json

```diff
@@ -49,7 +49,7 @@
       ]
     }
   },
-  "description": "Requires that a device be registered for Autopilot in order to enroll in Intune. The BUILTIN - Device Enrollers group is excluded and can enroll devices without Autopilot.",
+  "description": "What is it: Requires that a device be registered in Autopilot before enrolling in Intune.\n Simeon Notes: Prevent unauthorized devices from being enrolled into your tenant. In order to enroll a device a hardware hash must be uploaded to Intune or the user enrolling the device must be a member of the Azure AD group 'BUILTIN - Device Enrollers'. Simeon's baseline allows members of the Azure AD group 'BUILTIN - Device Enrollers' or 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations' to enroll devices without being registered. Note, that a device that has been registered in Intune is considered compliant, even if it is not yet enrolled.\n Learn More: https://www.lieben.nu/liebensraum/2019/10/hidden-exceptions-to-conditional-access-mfa/",
```

Comment:
**What does this do?**
Requires that a device be registered in Autopilot by uploading a hardware hash before it can be enrolled in Intune. To bypass this requirement, a user must be a member of the Azure AD group 'Baseline - Device Enrollers'. 

**Why should you use this?**
This increases security by preventing unauthorized devices from being enrolled into your tenant. 

**What is the end-user impact?**
IT must register devices in Autopilot before a general user can enroll it.

**Learn more**
N/A @metamenos we need to find something that isn't a personal blog

Reply:
I couldn't find anything from Microsoft that explained this. We can just leave empty